shared/AppInsightsCore/src/JavaScriptSDK.Interfaces/IProcessTelemetryContext.ts

```diff
@@ -27,12 +33,12 @@ export interface IProcessTelemetryContext {
     /**
      * Gets the current core config instance
      */
-    getCfg: ()=> IConfiguration;
+    getCfg: () => IConfiguration;
 
     /**
      * Gets the named extension config
      */
-    getExtCfg: <T>(identifier: string, defaultValue?:T|any) => T;
+    getExtCfg: <T>(identifier: string, defaultValue?: T | any, mergeDefault?: GetExtCfgMergeType) => T;
```

Comment:
Moving some code that was repeated across the code bases into the central location.